presidio-anonymizer/presidio_anonymizer/entities/anonymizer_request.py

```diff
@@ -87,31 +63,14 @@ def __handle_anonymizers(self, data):
         """
         anonymizers = data.get("anonymizers")
         if anonymizers is not None:
-            for key, anonymizer_dto in anonymizers.items():
-                self.logger.debug(f"converting {anonymizer_dto} to anonymizer class")
-                anonymizer = self.__get_anonymizer(anonymizer_dto)
-                self.logger.debug(f"applying class {anonymizer} to {anonymizer_dto}")
-                anonymizer_dto["anonymizer"] = anonymizer
-                self._anonymizers[key] = anonymizer_dto
+            for key, anonymizer_json in anonymizers.items():
+                self.logger.debug(
+                    f"converting {anonymizer_json} to anonymizer config class")
+                anonymizer_config = AnonymizerConfig.from_json(anonymizer_json)
+                self._anonymizers_config[key] = anonymizer_config
 
     def __handle_text(self, data):
         self._text = data.get("text")
```

Comment:
all other functions have type hints, and doc strings, should this also have it

Reply:
return type yes, doctsring no (only on public methods).

Reply:
but this method does not exist anymore :) in next PR